frontend/app/services/SubscriptionService.scala

```diff
@@ -131,12 +133,29 @@ object SubscriptionService {
   }
 }
 
-class SubscriptionService(val tierPlanRateIds: Map[ProductRatePlan, String],
-                          val zuoraSoapService: ZuoraSoapService,
+class SubscriptionService(val zuoraSoapService: ZuoraSoapService,
                           val zuoraRestService: ZuoraRestService) extends AmendSubscription with LazyLogging {
 
   import SubscriptionService._
-  val MembershipProductType = "Membership"
```

Comment:
Constants should be [Capitalised](http://docs.scala-lang.org/style/naming-conventions.html)!